src/main.cpp

```diff
@@ -36,6 +37,7 @@ int main(int argc, char *argv[])
     qmlRegisterType<VolumeControl>("org.asteroid.settings", 1, 0, "VolumeControl");
     qmlRegisterType<TiltToWake>("org.asteroid.settings", 1, 0, "TiltToWake");
     qmlRegisterType<TapToWake>("org.asteroid.settings", 1, 0, "TapToWake");
+    qmlRegisterType<Process>("Process", 1, 0, "Process");
```

Comment:
If we were to do this here, we should just expose Process as part of org.asteroid.settings like the examples above rather than add a new module just for one class

Reply:
But If we want to add an API to start commands, qml-asteroid utils feels like the better place to put them so they can be used by other apps too.

I think this sort of "low level" APIs go against the principles of good QML where only declarative UI bits are done in QML and all the lower level aspects are done in C++ but I'm not particularly close mind about the QML philosophy and I think I could live with classes that savagely punch through the intended abstraction layers.